.github/workflows/tests.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         export GPU_ARCH_TYPE=cpu
         export GPU_ARCH_VERSION=''
 
-        ./.github/scripts/unittest.sh
+        ${CONDA_RUN} ./.github/scripts/unittest.sh
```

Comment:
This is not related to the issue at hand.  Nevertheless, this is the correct way to invoke a script using `pytorch/test-infra/.github/workflows/macos_job.yml@main`.  Otherwise, the system conda will be used instead of the one setup by `macos_job` and the `ci` env created by the script won't be clean up correctly

Reply:
Thanks @huydhn , does that mean we should use that for the rest of the jobs as well (including linux and windows)?

Reply:
We don't need this on Linux because the script is run inside Docker, so cleaning things up is not an issue there and you can use your own conda installation if needed.  

On the other hand, this might be needed on Windows, but I haven't seen any disk space issue there before, so maybe it's not that important there.